src/Renci.SshNet.Tests/Classes/BaseClientTest_Connected_KeepAliveInterval_NotNegativeOne.cs

```diff
@@ -57,7 +57,11 @@ protected override void Act()
             _client.KeepAliveInterval = _keepAliveInterval;
 
             // allow keep-alive to be sent a few times
+#if NETFRAMEWORK
```

Comment:
Meaningful to have conditional compilation for this?

Reply:
TBH I didn't have a better idea for this. We have the following test:

https://github.com/sshnet/SSH.NET/blob/8201be44094555b1588331d339c771f7940ec6ab/src/Renci.SshNet.Tests/Classes/BaseClientTest_Connected_KeepAliveInterval_NotNegativeOne.cs#L99-L102

Which checks if the message was sent three times, but `.NET 7` has better performance and is able to send four messages in the same time.